xarray/tests/test_dataarray.py

```diff
@@ -1904,9 +1904,9 @@ def test_drop_coordinates(self):
         assert_identical(actual, expected)
 
         with raises_regex(ValueError, 'cannot be found'):
-            arr.drop(None)
+            arr.drop('w')
```

Comment:
What happens if `None` is passed? My quick test suggest this test should still pass, is that right?

Reply:
None _technically_ works, but it's not a valid use case. ``hash(None)`` works, but ``isinstance(None, Hashable)`` returns False. Also, xarray uses None everywhere to mark optional parameters, and in this case it asks the user to explicitly provide a value.